### PR TITLE
Reordered $panel-primary-text in variables.less for SASS compatibility

### PR DESCRIPTION
**Pull request by skateman**
I'm working on the [sass version](https://github.com/patternfly/patternfly-sass) of Patternfly and I have some issues with the variable order. Reordering the variables in the LESS version is an easier solution than doing this algorithmically with a parser. Also Bootstrap uses variables ordered by declarations first.

### CODE REVIEW

File: less/variables.less [hunk 1]
```diff
@@ -196,11 +196,11 @@
 @padding-small-vertical:                                            @padding-base-vertical;
 @panel-danger-border:                                               @brand-danger;
 @panel-danger-heading-bg:                                           @brand-danger;
+@panel-primary-text:                                                @body-bg;
```
Comment (rhamilto) [inline less/variables.less]: @skateman, Can you move this variable up into the section that precedes the current section (move it to line 149)?  That way it maintains the pseudo organization we've already got in the file.

Comment (skateman) [inline less/variables.less]: sure! :)

File: less/variables.less [hunk 2]
```diff
@@ -146,6 +146,7 @@
 @pagination-border:                                                 #bbbbbb;
 @pagination-hover-bg:                                               #ededed;
 @panel-inner-border:                                                #cecdcd;
+@panel-primary-text:                                                @body-bg;
```
Comment (rhamilto) [inline less/variables.less]: D'oh.  My bad.  This line should actually go to line 164 preceded by  "// References variable declared in block above".

Comment (skateman) [inline less/variables.less]: @rhamilto done :)

**Pull request merged**